structs/xmlStructs.go

```diff
@@ -261,6 +261,10 @@ type VolumeStatusXML struct {
 					BlockSize  int    `xml:"blockSize"`
 					MntOptions string `xml:"mntOptions"`
 					FsName     string `xml:"fsName"`
+					// As of Gluster3.12 this shows filesystem type. Bug?
```

Comment:
I don't remember what I wanted to do with "fsName". 
Maybe just nice to know how many of which fstype are running... without an poper use case in mind.